package.json

```diff
@@ -41,6 +42,7 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "@vitejs/plugin-react": "^4.0.3",
+    "cross-env": "^7.0.3",
```

Comment:
We can remove cross env as well? Does build script work for you?

Reply:
don't think so, reason being! build script doesn't work for me on winOS without adding the cross-env directly before the NODE-OPTIONS.

Reply:
But I don't see it getting used in the scripts in package.json